cartographer/mapping/3d/submap_3d.h

```diff
@@ -92,9 +101,11 @@ class ActiveSubmaps3D {
   // Inserts 'range_data' into the Submap collection. 'gravity_alignment' is
   // used for the orientation of new submaps so that the z axis approximately
   // aligns with gravity.
-  std::vector<std::shared_ptr<const Submap3D>> InsertRangeData(
+  std::vector<std::shared_ptr<const Submap3D>> InsertData(
       const sensor::RangeData& range_data,
-      const Eigen::Quaterniond& gravity_alignment);
+      const Eigen::Quaterniond& gravity_alignment,
+      const Eigen::VectorXf& rotational_scan_matcher_histogram_in_gravity,
```

Comment:
Do you think it's a better idea to pass the histogram rotated into the local map frame for better readability?

Reply:
Would that avoid having to pass in local_from_gravity? Then yes.

Reply:
okay done